core/src/main/java/io/confluent/kafka/schemaregistry/rest/exceptions/Errors.java

```diff
@@ -25,10 +25,13 @@
 
   // HTTP 404
   public static final String SUBJECT_NOT_FOUND_MESSAGE = "Subject not found.";
+  public static final String SUBJECT_NOT_FOUND_FOR_SUBJECT_MESSAGE_FORMAT = "Subject '%s' not found.";
```

Comment:
I'm open to alternative names

Reply:
Perhaps just `SUBJECT_NOT_FOUND_MESSAGE_FORMAT`.  With the new message format, we should possibly remove the old message?